src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -1,82 +1,79 @@
+ {{#> form-control controlType="input" input="true" type="email" id="textInput6" name="textInput6" aria-label="email input field" aria-describedby="email-example" }}
 {{/form-control}}
-  {{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="email-example"'}}
-    @example.com
-  {{/input-group-text}}
+ {{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="email-example"'}}
+   @example.com
+ {{/input-group-text}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="username"' aria-label="@"}}
-    <i class="fas fa-at" aria-hidden="true"></i>
-  {{/input-group-text}}
-  {{#> form-control required="true" controlType="input" input="true" type="email" id="textInput7" name="textInput7" aria-invalid="true" aria-label="Error state username example" aria-describedby="username"}}
+ {{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="username"' aria-label="@"}}
+   <i class="fas fa-at" aria-hidden="true"></i>
+ {{/input-group-text}}
+ {{#> form-control required="true" controlType="input" input="true" type="email" id="textInput7" name="textInput7" aria-invalid="true" aria-label="Error state username example" aria-describedby="username"}}
 {{/form-control}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> input-group-text inputGroupTextType="span"}}
-{{#> check pf-c-check-type="checkbox" name="checkbox example" id="checkbox1" aria-label="checkbox for following text input"}}
-{{/check}}
-  {{/input-group-text}}
-  {{#> form-control controlType="input" input="true" type="text" id="textInput8" name="textInput8" aria-label="input with checkbox example" aria-describedby="checkbox1"}}
+ {{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="calendar"' aria-label="calendar" }}
+   <i class="fas fa-calendar-alt" aria-hidden="true"></i>
+ {{/input-group-text}}
+ {{#> form-control controlType="input" input="true" type="date" id="textInput9" name="textInput9" aria-label="Date input example" aria-describedby="calendar" }}
 {{/form-control}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> input-group-action input-group-action--modifier="pf-m-align-left" inputGroupTextType="label" for="textInput9" }}
-  <i class="fas fa-calendar-alt" aria-hidden="true"></i>
-  {{/input-group-action}}
-  {{#> form-control controlType="input" input="true" type="date" id="textInput9" name="textInput9" aria-label="Date input example" }}
+ {{#> form-control controlType="input" input="true" type="search" id="textInput11" name="textInput11" aria-label="search input example" aria-describedby="search" }}
 {{/form-control}}
+{{#> input-group-text inputGroupTextType="span" input-group-text--attributes='id="search"' aria-label="search" }}
```

Comment:
I'm wondering if you meant to use a `button` for the "search" icon, and `inputGroupTextType="span"` for the `<i class="fas fa-question-circle aria-hidden="true"></i>` line below (L78)?

Reply:
I think this is right. For the search it's just an icon, where as the question mark is for a tooltip. Is that what you are talking about?

Reply:
Oh ok gotcha. I guess just looking at it, I would expect the search icon to be in a `button` (that you can press along with pressing `enter` to trigger the search). The question icon could either be in a button like you have it, or a text input group element with the question icon itself being the thing that triggers the tooltip.